rsocket/RSocketServer.cpp

```diff
@@ -128,7 +128,10 @@ void RSocketServer::onRSocketSetup(
     throw result.error();
   }
   auto connectionParams = result.value();
-  CHECK(connectionParams.responder);
+  if (!connectionParams.responder) {
+    LOG(ERROR) << "Received invalid Responder. Dropping connection";
```

Comment:
I wasn't commenting on the exception (which I think is fine), I just meant maybe replacing the `LOG(INFO)` with `VLOG(n)`.